packages/core/src/Synchronizer.ts

```diff
@@ -86,16 +89,13 @@ export class Synchronizer extends EventEmitter {
                 sealed: false,
             },
         })
-        if (epochs.length > 1) {
-            throw new Error('Multiple unsealed epochs')
-        }
         this.defaultStateTreeLeaf = BigInt(0)
         this.stateTree = new IncrementalMerkleTree(
             this.settings.stateTreeDepth,
             this.defaultStateTreeLeaf
         )
         // if it's a new sync, start with epoch 1
```

Comment:
description can be updated as well

Reply:
What do you mean? This error no longer exists, it's expected that the client has multiple unsealed epochs. This happens if there is no activity for multiple epochs. The next action will trigger an epoch transition across multiple epochs.

The error is completely removed in this PR.

Reply:
I mean this description line
https://github.com/Unirep/Unirep/blob/52cfb42b99f7b16a2cc88c7b89bfcf933b9113ec/packages/core/src/Synchronizer.ts#L97
Since the epoch updates from `0`

Reply:
ok [72ce291](https://github.com/Unirep/Unirep/pull/165/commits/72ce29162c972b3231367d858c4bad87e45460de)